code/modules/mob/mob_movement.dm

```diff
@@ -112,6 +112,10 @@
 		if(L.incorporeal_move)	//Move though walls
 			Process_Incorpmove(direct)
 			return 0
+	if(iscarbon(mob))
+		var/mob/living/carbon/C = mob
+		if(C.remote_view)

```

Comment:
I mean.. Y-y-you c-could use t-the remote_control var I added with the chronosuit just for things like this~